okon_actions_example.py

```diff
@@ -0,0 +1,60 @@
+""" Script for showing usage of actions defined in okon_acions module.
+
+Script prepared for testing actions defined in okon_actions module
+in OKON simulation (simulation-2.3dev version).
+
+Run this script after starting the simulation.
+"""
+import time
+
+import py_trees
+
+from okon_actions import Rotate, SetDepth
+from okon_client import OkonClient
+
+oc = OkonClient(ip="127.0.0.1", port=44210, sync_interval=.05, debug=False)
+oc.connect()
+time.sleep(1.)
+
+def create_root():
+    """ description
+
+    Long description
+    """
```

Comment:
I think we left this comment when I described docstrings to You. Consider changing it 😄